app/code/Magento/Sales/Model/Order/Invoice.php

```diff
@@ -502,7 +502,7 @@ public function getItemsCollection()
                 }
             }
         }
-        return $this->getItems();
+        return $this->getData(InvoiceInterface::ITEMS);
```

Comment:
@antonkril Here is what I changed here. 
Could you check this and tell me about getItems() method. I guess nothing should be changed there? Or I misunderstand something about your comment "Changing getItems() to hashmap .. "

Reply:
`getItems` is in the title of the issue, so we need to unify its behavior. Also, we need to modify `getItemsCollectionX` to not `setItems`

Reply:
Alright, but the behavior would be of the method getItemsCollection in this case? Of course it should return ItemsCollection as expected from the name. As Collection is iterable it will contain the items? But items could be empty and what to return in that case?